web/pages/api/profile/index.js

```diff
@@ -0,0 +1,18 @@
+import nextConnect from 'next-connect';
+import { query } from '../../../utils/query';
+
+const handler = nextConnect().get(async (req, res) => {
+	const user = req.user;
+
+	try {
+		const id = user.id;
+		var [results] = await query(`SELECT * FROM PROFILE WHERE userId = ?`, [id]);
+		if (results.length < 1) res.status(401).json({ error: id });
+
+		return res.json({ profile: results });
```

Comment:
This will return an array of results when we just want one. Maybe you meant `{profile: results[0]}`?